File: core/src/main/java/mifos/org/mobilewallet/core/domain/model/Currency.java
package mifos.org.mobilewallet.core.domain.model;

/**
 * Created by naman on 17/8/17.
 */

public class Currency {

    private String code;
    private String displaySymbol;
    private String displayLabel;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getDisplaySymbol() {
        return displaySymbol;
    }

    public void setDisplaySymbol(String displaySymbol) {
        this.displaySymbol = displaySymbol;
    }

    public String getDisplayLabel() {
        return displayLabel;
    }

    public void setDisplayLabel(String displayLabel) {
        this.displayLabel = displayLabel;
    }
}

File: core/src/main/java/org/mifos/mobilewallet/core/domain/model/Currency.kt
package org.mifos.mobilewallet.core.domain.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by naman on 17/8/17.
 */
@Parcelize
data class Currency(
        var code: String? = null,
        var displaySymbol: String? = null,
        var displayLabel: String? = null) : Parcelable